packages/fx-core/templates/plugins/resource/keyvault/bicep/keyVaultProvision.template.bicep

```diff
@@ -27,7 +28,7 @@ resource keyVault 'Microsoft.KeyVault/vaults@2019-09-01' = {
       }
     ]
     sku: {
-      name: 'standard'
+      name: keyVaultSkuName // You can follow https://aka.ms/teamsfx-bicep-add-param-tutorial to add "keyVaultSkuName" property to provisionParameters to override the default value "standard".
       family: 'A'
```

Comment:
should we also expose the family?

Reply:
family only has 'A' option. So no need to expose.